Optimize imports and fix NameError in static/ppdet/modeling/tests/test_architectures.py

Mandroide opened the pull request:
This cleanup unnecessary imports and solves the following exception:

![image](https://user-images.githubusercontent.com/31109774/124052497-8746b780-d9ec-11eb-94c8-0a4168d49e8e.png)

yghstill commented:
@Mandroide Good job, but these modifications have caused bugs, which can be modified according to the CI prompts.

Mandroide commented:
> @Mandroide Good job, but these modifications have caused bugs, which can be modified according to the CI prompts.

What are the CI prompts? I thought I could look at them on a webpage.

yghstill commented on ppdet/modeling/mot/tracker/base_jde_tracker.py:
```diff
@@ -18,7 +18,6 @@
 import numpy as np
 from collections import deque, OrderedDict
 from ..matching import jde_matching as matching
-from ppdet.core.workspace import register, serializable
```
Must keep `register` here. You can test  whether the `Training/Eval/Infer` can run successfully.

Mandroide replied:
Those two have been kept since they are decorators.

yghstill commented on deploy/python/infer.py:
```diff
@@ -27,7 +25,7 @@
 from paddle.inference import create_predictor
 
 from benchmark_utils import PaddleInferBenchmark
-from preprocess import preprocess, Resize, NormalizeImage, Permute, PadStride
```
Thank you for your modification,  The `Resize .etc` import here can't be deleted.
```
Traceback (most recent call last):
  File "deploy/python/infer.py", line 632, in <module>
    main()
  File "deploy/python/infer.py", line 599, in main
    predict_image(detector, img_list, FLAGS.batch_size)
  File "deploy/python/infer.py", line 511, in predict_image
    results = detector.predict(batch_image_list, FLAGS.threshold)
  File "deploy/python/infer.py", line 133, in predict
    inputs = self.preprocess(image_list)
  File "deploy/python/infer.py", line 96, in preprocess
    preprocess_ops.append(eval(op_type)(**new_op_info))
  File "<string>", line 1, in <module>
NameError: name 'Resize' is not defined
```
You can test `deploy/python/infer`:
```
python tools/export_model.py -c configs/yolov3/yolov3_darknet53_270e_coco.yml --output_dir=./inference_model -o weights=https://paddledet.bj.bcebos.com/models/yolov3_darknet53_270e_coco.pdparams

python deploy/python/infer.py --model_dir=./inference_model/yolov3_darknet53_270e_coco --image_file=./demo/road554.png --use_gpu=True
```

yghstill commented on static/ppdet/modeling/roi_extractors/roi_extractor.py:
```diff
@@ -19,7 +19,6 @@
 import paddle.fluid as fluid
 
 from ppdet.core.workspace import register
-from ppdet.modeling.ops import RoIAlign, RoIPool
```
Don't delete RoIAlign and RoIPool.  You can test static train/eval/infer

yghstill commented on static/ppdet/modeling/target_assigners.py:
```diff
@@ -19,7 +19,6 @@
 from paddle import fluid
 
 from ppdet.core.workspace import register
-from ppdet.modeling.ops import BBoxAssigner, MaskAssigner
```
Don't delete BBoxAssigner, MaskAssigne, please modify again, thx~

Mandroide committed:
Optimize imports

Besides removing unnecessary imports, this also fixes the missing logger module in test_architectures.py within the static directory.

yghstill reviewed (approved):
LGTM, at the same time, please cherry-pick this PR to `develop` branch.
